src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -76,14 +76,16 @@ public static class BitOperations
         /// </summary>
         /// <param name="value">The value.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static bool IsPow2(nint value) => (value & (value - 1)) == 0 && value > 0;
+        [CLSCompliant(false)]
```

Comment:
This shouldn't need `CLSCompliant(false)`, `IntPtr` itself is compliant.

Reply:
Uhm the build fails without this attribute :neutral_face:.
Error: CS3001

```
E:\dotnet\runtime>build.cmd
  Wiederherzustellende Projekte werden ermittelt...
  Das Tool "coverlet.console" (Version 1.7.2) wurde wiederhergestellt. Verfügbare Befehle: coverlet
  Das Tool "dotnet-reportgenerator-globaltool" (Version 4.5.8) wurde wiederhergestellt. Verfügbare Befehle: reportgenerator
  Das Tool "microsoft.dotnet.xharness.cli" (Version 1.0.0-prerelease.21451.1) wurde wiederhergestellt. Verfügbare Befehle: xharness
  Das Tool "microsoft.visualstudio.slngen.tool" (Version 6.1.10) wurde wiederhergestellt. Verfügbare Befehle: slngen

  Die Wiederherstellung war erfolgreich.
  Alle Projekte sind für die Wiederherstellung auf dem neuesten Stand.
  Wiederherzustellende Projekte werden ermittelt...
  Alle Projekte sind für die Wiederherstellung auf dem neuesten Stand.
  System.Private.CoreLib.Generators -> E:\dotnet\runtime\artifacts\bin\System.Private.CoreLib.Generators\netstandard2.0-Debug\System.Private.CoreLib.Generators.dll
E:\dotnet\runtime\src\libraries\System.Private.CoreLib\src\System\Numerics\BitOperations.cs(87,41): error CS3001: Argumenttyp "nuint" ist nicht CLS-kompatibel. [E:\dotnet\runtime\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]

Fehler beim Buildvorgang.

E:\dotnet\runtime\src\libraries\System.Private.CoreLib\src\System\Numerics\BitOperations.cs(87,41): error CS3001: Argumenttyp "nuint" ist nicht CLS-kompatibel. [E:\dotnet\runtime\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]
    0 Warnung(en)
    1 Fehler

Verstrichene Zeit 00:02:22.78
Build failed with exit code 1. Check errors above.
Some builds failed:
        Configuration: Debug, Architecture: x64
```

Reply:
That's complaining about L87 (the one taking `nuint`) not about L79 (the one taking `nint`).

Reply:
Thank you. I assumed that UIntPtr and IntPtr interact basically the same. 
Now the build is running for me.